packages/options/README.md

```diff
@@ -0,0 +1,25 @@
+[![npm][npm-image]][npm-url]
+[![npm-downloads][npm-downloads-image]][npm-url]
+[![semantic-release][semantic-release-image]][semantic-release-url]
+<br />
+[![code-style-prettier][code-style-prettier-image]][code-style-prettier-url]
+
+[code-style-prettier-image]: https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square
+[code-style-prettier-url]: https://github.com/prettier/prettier
+[npm-downloads-image]: https://img.shields.io/npm/dm/@solana/options/experimental.svg?style=flat
+[npm-image]: https://img.shields.io/npm/v/@solana/options/experimental.svg?style=flat
+[npm-url]: https://www.npmjs.com/package/@solana/options/v/experimental
+[semantic-release-image]: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+[semantic-release-url]: https://github.com/semantic-release/semantic-release
+
+# @solana/options
+
+This package allows us to manage and serialize Rust-like Option types in JavaScript. It can be used standalone, but it is also exported as part of the Solana JavaScript SDK [`@solana/web3.js@experimental`](https://github.com/solana-labs/solana-web3.js/tree/master/packages/library).
+
+## Types
+
+TODO
+
+## Functions
+
+TODO
```

Comment:
I'm guilty of this too. We need to catch all the READMEs up, soon.

Reply:
Yeah don't worry I'll make sure I tackle all these codec READMEs soon.